### PR TITLE
Bugfix/add vk link to profile

### PR DESCRIPTION
**Pull request by AVVasiliev**

### CODE REVIEW

File: visual_novel/core/commands.py [hunk 1]
```diff
@@ -48,3 +48,22 @@ def validate(self):
...
+
+    def __init__(self, data, user):
+        self.vk_link = data['vk_link']
+        self.user = user
+
+    def execute_validated(self):
+        profile = Profile.objects.get(user=self.user)
+        profile.vk_link = self.vk_link
+        profile.save()
+        # TODO: refresh subscriptions in Celery Task
```
Comment (dolamroth) [inline visual_novel/core/commands.py]: Remove comment

File: visual_novel/core/mixins.py [hunk 2]
```diff
@@ -1,6 +1,21 @@
 import datetime
 
-from .errors import WrongWeekdayBitmap, WrongIsSubscribed, WrongTime
+from notifications.vk import VK
+from .errors import WrongWeekdayBitmap, WrongIsSubscribed, WrongTime, WrongVkProfile
+
+
+class VkProfileValidator(object):
+    def check_vk_profile(self, vk_link):
+        # when user want delete link
```
Comment (dolamroth) [inline visual_novel/core/mixins.py]: Empty string corresponds to deleting a link

File: visual_novel/core/commands.py [hunk 3]
```diff
@@ -48,3 +48,22 @@ def validate(self):
         self.weekmap = self.validate_weekday_is_correct(self.weekmap)
         self.is_subscribed = self.validate_is_subscribed_field(self.is_subscribed)
         self.time = self.validate_time_field(self.time)
+
+
+class ChangeUserVkLinkOption(VkProfileValidator, Command):
+    """
+    TODO :raises WrongIsSubscribed: Raises if "vk_link" parameter has incorrect value.
```
Comment (dolamroth) [inline visual_novel/core/commands.py]: Поменять на 
raises WrongIsSubscribed: Raises if user with login equal to specified "vk_link" does not exist.

File: visual_novel/core/commands.py (hunk 1 above)
Comment (AVVasiliev) [reply visual_novel/core/commands.py]: check it

File: visual_novel/core/mixins.py (hunk 2 above)
Comment (AVVasiliev) [reply visual_novel/core/mixins.py]: fix

**Pull request merged**